### ISSUE TITLE
pull error

### ISSUE THREAD

**Issue by evalieungh** [NordicESMhub/docker-fates#1]
Hi! 
I tried to test the code in the readme file, but get an error in the beginning:
`docker pull quay.io/nordicesmhub/docker-fates:2.0.1`
gives error, also after logging in to my Docker account:
`Error response from daemon: unauthorized: access to the requested resource is not authorized`

Do we need any special access, or am I just doing it wrong?

**Comment by evalieungh** [comment]
there are no results for "docker-fates" on Docker hub. Is it set to private, perhaps?

**Comment by annefou** [comment]
Yes. sorry I forgot about that. When creating a new container in quay it is private by default. I changed it.

Anyway, this example is to show you how you can use quay.io/biocontainers/fates-emerald:2.0.1 (this one is always public) and how you can create your own container (for instance for your own cases). See Dockerfile in this repo.
 
Also better to "ping" me (@annefou) when creating issues (I am not monitoring all my repos).
